ApplicationCode/ProjectDataModel/Measurement/RimMeasurement.cpp

```diff
@@ -116,7 +115,6 @@ std::vector<cvf::Vec3d> RimMeasurement::pointsInDomainCoords() const
 void RimMeasurement::removeAllPoints()
 {
     m_pointsInDomainCoords.clear();
-    updateView();
```

Comment:
Is it corredt to remove this update ?

Reply:
The goal was to have a better separation of concerns, but the changeset will be smaller (and safer) if the call to updateView is kept. I will put it back.